staging/src/k8s.io/client-go/pkg/api/conversion.go

```diff
@@ -26,6 +26,7 @@ import (
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	selectors "k8s.io/apimachinery/pkg/selectors"
```

Comment:
I added it to break a circular dependency I created when I had labels depend on StatusError.